test/connectors/corrections/CorrectionConnectorSpec.scala

```diff
@@ -77,4 +77,40 @@ class CorrectionConnectorSpec extends SpecBase with WireMockHelper with EitherVa
       }
     }
   }
+
+  ".getForCorrectionPeriod" - {
```

Comment:
can we add a scenario or two to cover Left response

Reply:
added 2 scenarios